plugins/modules/aci_endpoint_ip_tag.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_endpoint_ip_tag
+short_description: Manage Endpoint IP Tag (fv:EpIpTag)
+description:
+- Manage Endpoint IP Tag on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+  endpoint_ip_address:
```

Comment:
```suggestion
  ip:
```
Swap endpoint_ip_address and ip as primary

Reply:
Done